src/DynamoAnalyzer/Helper/GregHelper.cs

```diff
@@ -0,0 +1,241 @@
+using DynamoAnalyzer.Models;
+using Greg;
+using Greg.Requests;
+using Greg.Responses;
+
+namespace DynamoAnalyzer.Helper
+{
+    /// <summary>
+    /// Provides methos to manage packages
+    /// </summary>
+    public static class GregHelper
+    {
+        private static GregClient _client;
+
+        public static GregClient GetClient()
+        {
+            return _client ??= new GregClient(new Dynamo.Core.IDSDKManager(), ConfigHelper.GetConfiguration()["DynamoPackagesURL"]);
```

Comment:
you don't need `IDSDKManager` - thats only required for auth, you can just make public rest requests to package manager. I think you can get rid of this reference to core.

Reply:
you can also remove `DynamoPackagesURL` - thats already defined in the DynamoPackages.dll.config file.

Reply:
I have removed the GregClient dependency by a RestSharp request